tests/LangCheckTest.php

```diff
@@ -1,7 +1,10 @@
 <?php
+
+use PHPUnit\Framework\TestCase;
+
 require_once __DIR__ . '/LangCheck.php';
 
-class LangCheckTest extends PHPUnit_Framework_TestCase
+class LangCheckTest extends TestCase
```

Comment:
What necessitates the rename of this class?

Reply:
Newer phpunits use namespaced classes instead of the prefix.

Reply:
Shouldn't it than use something along the lines of https://stackoverflow.com/a/42828632 for BC?

Reply:
I'm not sure, but I think we would need backward compatibility  for PHP < 5.6 only. All newer PHP versions should work with namespaced versions of phpunit.

Reply:
As compat is still at least PHP 5.3 (or even older) if we don't intentionally break things, we shouldn't break it for PHPUnit here either.